docs/explanation/main-inclusion-review.rst

```diff
@@ -0,0 +1,74 @@
+Main Inclusion Review (MIR)
+===========================
+
+.. important::
+
+    Do not confuse the abbreviation :term:`MIR` with the `display server <https://mir-server.io/>`_.
+
+:term:`Packages <Package>` in :term:`Main` and :term:`Restricted` are officially
+maintained, supported and recommended by the :term:`Ubuntu` project.
+:term:`Canonical's <Canonical>` support services applies to these :term:`Packages <Package>`, 
+which include security updates and certain :term:`SLA` grantees when bugs are reported
+and technical support is requested.
+
+Therefore, special consideration is necessary before adding new :term:`Packages <Package>`
+to :term:`Main` or :term:`Restricted`. The :term:`Ubuntu` :term:`MIR Team` reviews
+:term:`Packages <Package>` for promotion
+
+- from :term:`Universe` to :term:`Main`
+- or from :term:`Multiverse` to :term:`Restricted`.
+
+This review process is called *Main Inclusion Review (MIR)*.
+
+Apply a package for Main Inclusion Review
```

Comment:
This is a slightly odd ordering of words, although I think I know what you meant. I'd suggest either "Main Inclusion Review package application" or "Submit a package for Main Inclusion Review". You could also have "Apply for Main Inclusion Review for a package", although that's slightly longer.

Reply:
Now that mention it I see it too.

Reply:
Resolved with 89bec8a88797da66ab13f3414a42b5e33e2670fd